website/src/components/SearchPage/DownloadDialog/DownloadUrlGenerator.ts

```diff
@@ -86,4 +100,21 @@ export class DownloadUrlGenerator {
         const timestamp = new Date().toISOString().slice(0, 16).replace(':', '');
         return `${organism}_${dataType}_${timestamp}`;
     }
+
+    private downloadEndpoint(dataType: DownloadDataType) {
+        const segmentPath = (segment?: string) => (segment !== undefined ? '/' + segment : '');
+
+        switch (dataType.type) {
+            case 'metadata':
+                return this.lapisUrl + '/sample/details';
+            case 'unalignedNucleotideSequences':
+                return dataType.includeRichFastaHeaders === true
+                    ? '/' + this.organism + '/api/sequences'
```

Comment:
This should be a full URL because the value is also used when a user uses the "Copy download URL" button.